_data/versions.yml

```diff
@@ -219,6 +219,29 @@ v3.9:
       version: v3.9.0
 
 v3.8:
+- title: v3.8.6
+  components:
+     typha:
+      version: v3.8.6
+     calicoctl:
+      version: v3.8.6
+     calico/node:
+      version: v3.8.6
+     calico/cni:
+      version: v3.8.6
+     calico/kube-controllers:
+      version: v3.8.6
+     networking-calico:
+      version: 3.8.6
+     flannel:
+      version: v0.11.0
+     calico/dikastes:
+      version: v3.8.6
+     flexvol:
+      version: v3.8.6
+     calico/flannel-migration-controller:
```

Comment:
Should this be here?  Or will we just ignore it?

Reply:
Oops, yes I need to fix this too

Reply:
d8e7f1c2bdb603a818c1d6acf853346e53efacbf